### ISSUE TITLE
Provide incentives for users to play Qualified maps

### ISSUE THREAD

**Issue by Ephemeralis** [ppy/osu-web#10154]
Put shortly, Qualified maps are not being played in sufficient volumes to serve the category's primary purpose, aka vetting and playtesting Ranked-candidate maps in real-world conditions. The reason for this is very simple: players get nothing out of it. 

Engaging for a map for the first time in Qualified has an opportunity cost involved, as any scores you make are wiped before ranking, and the nature of players replicating scores means that there may be a great deal of frustration involved in doing it again. The obvious (and oft-suggested) solution to this is to retain Qualified scores into Ranked, but this then creates societal pressure for players to _not_ report issues for fear of having their scores removed in the process of fixing things.

I'd like to suggest that we trial incentives in Qualified in the following three ways to start:

- A new "Qualified Score" metric is added to user profiles. This tracks the total "ranked score" of any plays made on Qualified maps. For the sake of simplicity, this is only ever incremented upwards and never otherwise adjusted to reflect changes made to existing Qualified maps. You play a Qualified map, your score goes up. Simple as.
- Using Qualified Score as suggested above, create two "leaderboard" views on the site (for each game mode): one showing all-time Qualified Score, and one showing the current month's Qualified Score tally for all users.
- We can then run monthly supporter prizes for the top N Qualified Score players in each game mode, and track these metrics for things like medals, achievements, etc. The NAT have a ton of ideas we can try so long as this metric exists for us to play with.

The end goal is to foster a small community of people willing to play Qualified maps and then hoping that an even smaller percentage of these individuals ultimately choose to engage in beatmap discussions to report any issues they encounter.

**Comment by tailsdk** [comment]
I don't think this will really get more people participating in discussions. It will probably get more people playing, but there is quite a big jump from playing to commenting on the beatmap discussion. Simply because most people who play have no idea how modding works. I think if the goal is to have more discussion on the qualified section a better way might be giving some sort of reward to the people that check them. Currently there is very little incentive to check qualified maps because it gives very little in return. I also think that giving rewards might incentivise smaller non issue posting, where its not really a problem but could be changed like adding a single tag and stuff.

**Comment by Ephemeralis** [comment]
The idea is to increase the volume of people engaging with Qualified maps with incentives, so that the increased numbers equates to increased vigilance on any potential issues the maps may have.

As a few people have demonstrated, it only takes one person to report a problem to potentially address it.

**Comment by Hiviexd** [comment]
how about we just let people gain ranked score from qualified maps and keep said score when said map gets ranked, while keeping pp gain from it at a zero? ranked score loss from dqs is marginally less impactful than pp loss from dqs, and with the warnings we already have, the issue of people getting mad over dqs wouldn't really be a detriment in this case.

**Comment by cl8n** [comment]
this seems like a lot of investment required to build a thing that only maybe will achieve the result you're looking for. I feel like it's more suited to start as a third-party service if something similar doesn't exist already (thought osu!alternative might have it but it looks like they aren't tracking qualified rn)

besides that, I'm not a fan of giving "incentive" to gameplay that people are not widely enjoying as-is. I feel like everyone winning the supporter prizes for example are just gonna play until they have a placement they're happy with and then be glad that they're done, just like the participation pattern of later monthly charts and early spotlights, which this proposal is very reminiscent of. it does not sound appealing to play other than as an out-of-the-way grind for whatever rewards you're offering
